Dockerfile

```diff
@@ -165,7 +166,8 @@ RUN sed -i 's|http://archive.ubuntu.com|http://us.archive.ubuntu.com|g' /etc/apt
         # Add ROS 2 repo
         apt-key adv --fetch-keys https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc && \
         sh -c 'echo "deb [arch=$(dpkg --print-architecture)] http://packages.ros.org/ros2/ubuntu $(grep -oP "UBUNTU_CODENAME\=\K.*" /etc/os-release) main" > /etc/apt/sources.list.d/ros2-latest.list' && \
-        apt-get update && \
+        # Install Sonar dependency nodejs
+        curl -sL https://deb.nodesource.com/setup_16.x | sudo bash - && \
```

Comment:
Is there a reason we need this curl installation if we've already got nodejs as an apt package from line 85? Is this pulling in a different version of node?

Reply:
Oddly enough the setup script does everything except install nodejs, it's still necessary to `apt-get install nodejs` after the setup script has added the repo. You can see the script here to confirm that if you'd like: https://deb.nodesource.com/setup_16.x